cmd/root.go

```diff
@@ -111,6 +117,60 @@ func args() []string {
 	return args
 }
 
+func bugReport(ctx context.Context) error {
+	var containerDaemonPaths = []string{
+		"/var/run/docker.sock",
+		"/var/run/podman/podman.sock",
+		"$HOME/.colima/docker.sock",
```

Comment:
I guess this could also be: `$XDG_RUNTIME_DIR/docker.sock` in case of rootless docker, right? https://docs.docker.com/engine/security/rootless/#daemon

Reply:
But nevertheless, rootless docker also needs `DOCKER_HOST` to be set